WalletWasabi/WabiSabi/Client/CoinJoin/Client/CoinJoinClient.cs

```diff
@@ -152,6 +155,12 @@ public async Task<CoinJoinResult> StartCoinJoinAsync(Func<Task<IEnumerable<Smart
 					currentRoundState.LogInfo(roundSkippedMessage);
 					throw new CoinJoinClientException(CoinjoinError.UneconomicalRound, roundSkippedMessage);
 				}
+				if (roundParameters.CoordinationFeeRate.Rate * 100 > MaxAllowedCoordinationFeeRate)
+				{
+					string roundSkippedMessage = "Coordination fee rate was too high.";
```

Comment:
I think the message should metnion both value. And probably it should also mention that they can change it in the setting (but probably the message would be too long)

Reply:
How about popping a dialog that "We detected this coordinator charges 0.1% of coordinator fee, Do you accept it?" Yes/No

Of course, only pop this when the coordinator fee > MaxAllowedCoordinationFee

Reply:
With this, you would actually protect the user, but still, the decision is the user's.

Reply:
> I think the message should metnion both value. And probably it should also mention that they can change it in the setting (but probably the message would be too long)

This is not the message that appears in the `MusicBox`, this is only logged. I will add the values.

Reply:
There is a mistake, you shouldn't multiply it by 100.
If you set in WabiSabiConfig 0.003 per round and same in client you'll get an error. You'll need to set 0.3% in client per round to make it work.
No need to *100. Please check.

Reply:
@turbolay please doublecheck.

Reply:
There is no mistake. On server side config this is a rate, on the UI this is a percentage. If you set 0.003 on the server it means 0.003*100 = 0.3%

Do you think it is not clear?

Reply:
It is true that I could have named internally `Percent` instead of `Rate` but I think it is ok